xray/src/generation.rs

```diff
@@ -29,6 +79,9 @@ pub enum ColoringStrategyKind {
 
     // Min and max intensities.
     ColoredWithIntensity(f32, f32),
+
+    // Colored in heat-map colors by stddev. Takes the max stddev to clamp on.
+    ColoredWithHeightStddev(f32),
```

Comment:
Nit: ColoredWithHeightSD?

Reply:
I kept it since stddev seems more common to me.